pages/listings.tsx

```diff
@@ -1,7 +1,69 @@
-export default function ListingsPage() {
+import AuthGuard from "@App/lib/auth/AuthGuard";
+import useAuthContext from "@App/lib/auth/AuthContext";
+import fetchCurrentUserListings from "@App/lib/listings/useFetchCurrentUserListing";
+import { Card, CardMedia, Stack, CardContent, Box, LinearProgress, Paper, Typography, Button } from "@mui/material";
+import { updateCurrentUser } from "firebase/auth";
+import type { NextPage } from "next";
+import {TListingCollectionReference} from "@App/lib/listings/types"
+
+const ListingsPage: NextPage = () => {
+
+  const { currentUser } = useAuthContext()
+  let obj:TListingCollectionReference;
+  obj={userId: currentUser?.id!}
```

Comment:
this let variable isn't necessary. you should just pass the object into the hook